src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -65,6 +65,9 @@ protected void onCreate(Bundle savedInstanceState) {
                     throw new IllegalArgumentException(
                             "You must provide the requested fragment type as an extra.");
             }
+        } else {
+            setTitle(R.string.settings_title);
+            mFragment = new SettingsFragment();
```

Comment:
Yes they are, because Log is an activity and has a 'return' behavior to go to SettingsActivity. Therefore it will be started with an empty action intent.